GridMaker_Sketch.py

```diff
@@ -11,12 +13,22 @@ def __init__(self, **entries):
 
 
 def readJsonFile(file):
+    if not file:
+        raise ValueError("File not found or file pattern did not match.")
```

Comment:
I don't think this will work, currently you're checking is a String param is true and throwing an exception if it isn't. If you want some safety, we can wrap the `with open(file, 'r') as openFile:` block in some try catch code, print a nice message. and do a `sys.exit()` in the catch